<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TH1716" ignored="5" name="Test12" tests="1" failures="1" timestamp="17 Aug 2018 12:20:22 GMT" time="198.776" errors="0">
  <testcase name="login_test1" time="198.757" classname="login_multiuser.MultiUser_Test">
    <failure type="org.openqa.selenium.WebDriverException" message="java.net.SocketException: Connection reset
Build info: version: &amp;apos;3.8.1&amp;apos;, revision: &amp;apos;6e95a6684b&amp;apos;, time: &amp;apos;2017-12-01T18:33:54.468Z&amp;apos;
System info: host: &amp;apos;TH1716&amp;apos;, ip: &amp;apos;172.16.10.82&amp;apos;, os.name: &amp;apos;Windows 8.1&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;6.3&amp;apos;, java.version: &amp;apos;1.8.0_11&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.WebDriverException: java.net.SocketException: Connection reset
Build info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T18:33:54.468Z'
System info: host: 'TH1716', ip: '172.16.10.82', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '1.8.0_11'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:92)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:398)
at org.openqa.selenium.remote.RemoteWebDriver.findElementsByLinkText(RemoteWebDriver.java:429)
at org.openqa.selenium.By$ByLinkText.findElements(By.java:241)
at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:359)
at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:881)
at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:43)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
at org.openqa.selenium.support.ui.ExpectedConditions$22.apply(ExpectedConditions.java:643)
at org.openqa.selenium.support.ui.ExpectedConditions$22.apply(ExpectedConditions.java:640)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
at com.milvik.mip.pageutil.MIP_BasePage.waitForElementTobeClickable(MIP_BasePage.java:61)
at login_multiuser.MultiUser_Test.login_test1(MultiUser_Test.java:55)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
Caused by: java.net.SocketException: Connection reset
at java.net.SocketInputStream.read(Unknown Source)
at java.net.SocketInputStream.read(Unknown Source)
at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:137)
at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:153)
at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:282)
at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:138)
at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:56)
at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:259)
at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:163)
at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:165)
at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:273)
at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:125)
at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:272)
at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:89)
at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:111)
at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:72)
at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
at org.openqa.selenium.remote.internal.ApacheHttpClient.fallBackExecute(ApacheHttpClient.java:138)
at org.openqa.selenium.remote.internal.ApacheHttpClient.execute(ApacheHttpClient.java:86)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:161)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
... 34 more
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- login_test1 -->
  <testcase name="login_test2" time="0.0" classname="login_multiuser.MultiUser_Test">
    <ignored/>
  </testcase> <!-- login_test2 -->
  <testcase name="login_test3" time="0.0" classname="login_multiuser.MultiUser_Test">
    <ignored/>
  </testcase> <!-- login_test3 -->
  <testcase name="login_test6" time="0.0" classname="login_multiuser.MultiUser_Test">
    <ignored/>
  </testcase> <!-- login_test6 -->
  <testcase name="login_test4" time="0.0" classname="login_multiuser.MultiUser_Test">
    <ignored/>
  </testcase> <!-- login_test4 -->
  <testcase name="login_test5" time="0.0" classname="login_multiuser.MultiUser_Test">
    <ignored/>
  </testcase> <!-- login_test5 -->
</testsuite> <!-- Test12 -->
